src/DateRangePicker.jsx

```diff
@@ -6,12 +6,12 @@ import calendar from 'calendar';
 
 import BemMixin from './utils/BemMixin';
 import CustomPropTypes from './utils/CustomPropTypes';
-import Legend from './Legend';
+import Legend from './Legend.jsx';
```

Comment:
Why the addition of the extensions? We should be able to resolve for both `.jsx` and `.js` files